server/src/internalClusterTest/java/org/elasticsearch/action/admin/cluster/desirednodes/TransportDesiredNodesActionsIT.java

```diff
@@ -166,7 +172,7 @@ public void testUnknownSettingsAreAllowedInFutureVersions() {
         assertThat(latestDesiredNodes, is(equalTo(desiredNodes)));
     }
 
-    public void testSomeSettingsCanBeOverridden() {
+    public void testNodeProcessorsGetValidatedWithDesiredNodeProcessors() {
```

Comment:
I think this verifies what the method name says by not throwing when setting the desired nodes? Perhaps add a comment if so.